starboard/loader_app/BUILD.gn

```diff
@@ -319,7 +322,7 @@ if (sb_is_evergreen) {
       ":installation_manager",
       ":installation_store_proto",
       ":slot_management",
-      "//starboard:starboard_with_main",
+      "//starboard:starboard_with_main($starboard_toolchain)",
```

Comment:
Instead of doing this, add to the condition for building slot_management_test: `if (sb_is_evergreen && current_toolchain == starboard_toolchain) {`